src/script/event/preprocessor/QuoteDecoderMiddleware.test.ts

```diff
@@ -101,57 +100,5 @@ describe('QuotedMessageMiddleware', () => {
       expect(parsedEvent.data.quote.message_id).toEqual(quotedMessage.id);
       expect(parsedEvent.data.quote.user_id).toEqual(quotedMessage.from);
     });
-
-    it('updates quotes in DB when a message is edited', async () => {
-      const [quotedMessageMiddleware, {eventService}] = buildQuotedMessageMiddleware();
-      const originalMessage = toSavedEvent(createMessageAddEvent());
-      const replies = [
-        createMessageAddEvent({dataOverrides: {quote: {message_id: originalMessage.id} as any}}),
-        createMessageAddEvent({dataOverrides: {quote: {message_id: originalMessage.id} as any}}),
-      ];
-      eventService.loadEvent.mockResolvedValue(originalMessage);
-      eventService.loadEventsReplyingToMessage.mockResolvedValue(replies);
-
-      const event = createMessageAddEvent({dataOverrides: {replacing_message_id: originalMessage.id}});
-
-      jest.useFakeTimers();
-
-      await quotedMessageMiddleware.processEvent(event);
-      jest.advanceTimersByTime(1);
-
-      expect(eventService.replaceEvent).toHaveBeenCalledWith(
-        jasmine.objectContaining({data: jasmine.objectContaining({quote: {message_id: event.id}})}),
-      );
-      jest.useRealTimers();
-    });
-
-    it('invalidates quotes in DB when a message is deleted', () => {
-      const [quotedMessageMiddleware, {eventService}] = buildQuotedMessageMiddleware();
-      const originalMessage = toSavedEvent(createMessageAddEvent());
-      const replies = [
-        createMessageAddEvent({dataOverrides: {quote: {message_id: originalMessage.id} as any}}),
-        createMessageAddEvent({dataOverrides: {quote: {message_id: originalMessage.id} as any}}),
-      ];
-      spyOn(eventService, 'loadEvent').and.returnValue(Promise.resolve(originalMessage));
-      spyOn(eventService, 'loadEventsReplyingToMessage').and.returnValue(Promise.resolve(replies));
-      spyOn(eventService, 'replaceEvent').and.returnValue(Promise.resolve());
-
-      const event = {
-        conversation: 'conversation-uuid',
-        data: {
-          replacing_message_id: 'original-id',
-        },
-        id: 'new-id',
-        type: ClientEvent.CONVERSATION.MESSAGE_DELETE,
-      } as any;
-
-      return quotedMessageMiddleware.processEvent(event).then(() => {
-        expect(eventService.replaceEvent).toHaveBeenCalledWith(
-          jasmine.objectContaining({
-            data: jasmine.objectContaining({quote: {error: {type: QuoteEntity.ERROR.MESSAGE_NOT_FOUND}}}),
-          }),
-        );
-      });
-    });
```

Comment:
Those features are not covered by the new `RepliesUpdaterMiddleware`

Reply:
no**w** covered? :D

Reply:
oh yes 😅  I wrote exactly the opposite of what I meant 🙃